File: aisile-parent/aisile-manager-web/src/main/webapp/js/service/typeTemplateService.js
//封装所有连接后端的代码,需要$http
app.service('typeTemplateService',function($http){
		    	//读取列表数据绑定到表单中
				//查询全部
		    	this.findAll=function(){
		    		return $http.get('../typeTemplate/findAll.do');		
		    	}
		    	//分页查询
		    	this.findPage=function(page,rows){
		    		return $http.get('../typeTemplate/findPage.do?page='+page+'&rows='+rows);		
		    	}
		    	//模糊分页查询
		    	this.findSearch=function(page,rows,searchEntity){
		    		return $http.post('../typeTemplate/findSearch.do?page='+page+'&rows='+rows,searchEntity);		
		    	}
		    	//修改
		    	this.update=function(entity){
		    		return $http.post('../typeTemplate/update.do',entity);		
		    	}
		    	//添加
		    	this.add=function(entity){
		    		return $http.post('../typeTemplate/add.do',entity);		
		    	}
		    	//根据id查询实体对象
		    	this.findOne=function(id){
		    		return $http.get('../typeTemplate/findOne.do?id='+id);		
		    	}
		    	//删除方法
		    	this.dele=function(selectIds){
		    		return $http.get('../typeTemplate/delete.do?ids='+selectIds);		
		    	}
		    	
}); 

File: aisile-parent/aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/TypeTemplateService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbSpecification;
import com.aisile.pojo.TbTypeTemplate;
import com.aisile.pojo.entity.PageResult;
import com.aisile.pojo.group.Specification;

public interface TypeTemplateService {
	//查询全部规格的方法
	public List<TbTypeTemplate> findAll();
	
	//分页方法
	public PageResult findPage(int pageNum,int pageSize);
	
	//模糊分页方法
	public PageResult findSearch(int pageNum,int pageSize,TbTypeTemplate tbTypeTemplate);
	
	//添加方法
	public void add(TbTypeTemplate tbTypeTemplate);
		
	//修改方法
	public void update(TbTypeTemplate tbTypeTemplate);
		
	//根据id获取实体对象
	public TbTypeTemplate findOne(Long id);
		
	//批量删除
	public void delete(Long [] ids);
	
	//查询规格列表方法
	public List<Map> findOptionsList(Long id);
	
	
}
